tests/flow/test_reversed_patterns.py

```diff
@@ -1,46 +1,28 @@
 import os
 import sys
-import unittest
 from redisgraph import Graph, Node, Edge
 
-# import redis
 sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
```

Comment:
is this `sys.path.append` still necessary?

Reply:
done